### PR TITLE
Added 'prettier' and 'pre-commit' dependencies. Other minor changes.

### PR DESCRIPTION
**Pull request by PauloGoncalvesBH**
#### Package.json file
- Added 'prettier' dependency to prettify the code. (https://github.com/prettier/prettier).
   - Created script 'prettify' on package.json to run the prettier.
   - Will prettify:
      - All '.js' and '.md' file on root and subfolder.
      - package.json.
      - All files inside sampleApp folder (which follow the prettier parameters).
- Added the 'pre-commit' dependency.
   - Configured to script 'prettify' execute on pre-commit.

- Moved the command 'webdriver-manager update' from 'test' script to 'pretest' to better readability.
#### README.md file
- Added new badges:
   - NPM Weekly Downloads.
   - Code Style: Prettier.
   - Build Status.

#### Other files:
- Changes made by the script 'prettify' on pre-commit.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,8 +2,13 @@
   "name": "protractor-helper",
   "version": "3.6.13",
   "scripts": {
-    "test": "webdriver-manager update && protractor test/protractor.conf.js"
+    "pretest": "webdriver-manager update",
+    "test": "protractor test/protractor.conf.js",
+    "prettify": "prettier --write **/*.js **/*.md package.json sampleApp/**/*"
```
Comment (wlsf82) [inline package.json]: Will files inside `constants_and_utils` directory be prettified as well with this config?

Comment (PauloGoncalvesBH) [reply package.json]: Yes, when the `prettify` runs, the following ocurrs:

image

The script `prettify` is `prettier --write **/*.js **/*.md package.json sampleApp/**/*`.

 Explaining him, the `**/*.js` command find all javascript files in the same directory and subfolder:
 ```
  constant_and_utils\constants.js
  constant_and_utils\utils.js
  index.js
  test\constants.js
  test\protractor.conf.js
  test\sample.spec.js
 ```

 The same ocurrs with `.md`, finding all markdoowns in the same directory and subfolder.
```
.github\ISSUE_TEMPLATE.md
.github\PULL_REQUEST_TEMPLATE.md
CHANGELOG.md
docs\CONTRIBUTING.md
docs\EXAMPLES.md
README.md
```

The `package.json` is self explanatory.

The `sampleApp/**/*` find all files inside sampleApp folder and all files inside the subfolder on sampleApp.
```
sampleApp\index.html
sampleApp\public\style.css
```
## Explain 2 asterisks
 The follow comand find javascript files only in any subfolder.
`*/*.js`

 The follow comand with 2 asterisks find all javascript files in the same directory and in any subfolder.
`**/*.js`

 That is, the `**/*.js` command is equal the following two commands: `*.js */*.js`.

Comment (PauloGoncalvesBH) [reply package.json]: A bit more information about prettier:
 When he format any file, the color com command line change from grey to white.
 Example formatting the package.json file:

image

Comment (wlsf82) [reply package.json]: Cool. Thanks!

**Pull request merged**